adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -620,7 +620,9 @@ public Intent getIntentForBrokerActivity(final AuthenticationRequest request, fi
             intent = BrokerAccountServiceHandler.getInstance().getIntentForInteractiveRequest(mContext, brokerEvent);
             if (intent == null) {
                 Logger.e(TAG, "Received null intent from broker interactive request.", null, ADALError.BROKER_AUTHENTICATOR_NOT_RESPONDING);
-                throw new AuthenticationException(ADALError.BROKER_AUTHENTICATOR_NOT_RESPONDING, "Received null intent from broker interactive request.");
+                final AuthenticationException authenticationException = new AuthenticationException(ADALError.BROKER_AUTHENTICATOR_NOT_RESPONDING, "Received null intent from broker interactive request.");
+                brokerEvent.setBrokerAccountServiceConnectionErrorInfo(authenticationException);
```

Comment:
>setBrokerAccountServiceConnectionErrorInfo [](start = 28, length = 42)

what would this look like - example for Bound Service case?

Reply:
Ashish and I spoke this morning about reducing the size of stack traces that this will produce. I'll post a sample of that data as soon as I have finished making the changes 👍